ruler-gradle-plugin/src/main/kotlin/com/spotify/ruler/plugin/apk/ApkCreator.kt

```diff
@@ -94,6 +98,27 @@ class ApkCreator(private val rootDir: File) {
         return SdkLocator.getSdkDirectory(rootDir, issueReporter).toPath()
     }
 
+    /**
+     * Grab the debug keystore when creating the split apks. Doing this step makes sure
+     * the corresponding /META-INF/BNDLTOOL.SF and *.RSA files are created in the apks.
+     **/
+    private fun getAndroidDebugKey(): SigningConfiguration {
+        val androidDebugKey = File(System.getProperty("user.home"), ".android/debug.keystore")
```

Comment:
I think this is fine, maybe having a possibility to configure the path would be a good idea.